core/tauri/src/manager/webview.rs

```diff
@@ -534,6 +534,23 @@ impl<R: Runtime> WebviewManager<R> {
       }
     }
 
+    #[cfg(not(target_os = "windows"))]
```

Comment:
Shouldn't be injected on mobile platforms
```suggestion
    #[cfg(all(desktop, not(target_os = "windows")))]
```